src/reducers/admin.js

```diff
@@ -62,11 +63,26 @@ export default function reducer(state = initialState, action) {
         ...state,
         adminAddress: action.payload
       };
+    case types.LOADING_ADMIN_ADDRESS_FAILURE:
+      return {
+        ...state,
+        adminAddress: ""
+      };
     case types.DEPLOYING_STORE:
       return {
         ...state,
         deploying: true
       };
+    case types.LOADING_ACCOUNT_BALANCE_SUCCESS:
+      return {
+        ...state,
+        accountBalance: action.payload
+      };
+    case types.LOADING_ACCOUNT_BALANCE_FAILURE:
+      return {
+        ...state,
+        accountBalance: ""
```

Comment:
shouldnt be a float ? so set null | undefined ?

Reply:
Updated.